sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -175,13 +175,25 @@ def _get_statement_metrics_query_cached(self, cursor):
         return self._statement_metrics_query
 
     def _load_raw_query_metrics_rows(self, cursor):
+        start_time = time.time()
         self.log.debug("collecting sql server statement metrics")
         statement_metrics_query = self._get_statement_metrics_query_cached(cursor)
         self.log.debug("Running query [%s]", statement_metrics_query)
         cursor.execute(statement_metrics_query)
         columns = [i[0] for i in cursor.description]
         # construct row dicts manually as there's no DictCursor for pyodbc
         rows = [dict(zip(columns, row)) for row in cursor.fetchall()]
+        self._check.gauge(
+            "dd.sqlserver.statments.load_query_metrics_rows.rows",
+            len(rows),
+            **self._statsd_debug_kwargs(),
+        )
+        elapsed_ms = (time.time() - start_time) * 1000
+        self.check.histogram(
+            "dd.sqlserver.statements.collect_statement_metrics_and_plans.time",
```

Comment:
I'm guessing that's copy-pasta of the gauge for the overall collection? Shouldn't this be `dd.sqlserver.statements.load_statement_metrics.time` instead?

Reply:
updated to use https://github.com/DataDog/integrations-core/pull/10809